Vostok.Hosting/Components/Log/LogLevelMetrics.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using Vostok.Commons.Time;
+using Vostok.Hosting.Abstractions;
+using Vostok.Logging.Abstractions;
+using Vostok.Metrics;
+using Vostok.Metrics.Models;
+using Vostok.Metrics.Primitives.Gauge;
+
+namespace Vostok.Hosting.Components.Log
+{
+    internal class LogLevelMetrics
+    {
+        private readonly LogEventLevelCounter counter;
+
+        public LogLevelMetrics(LogEventLevelCounter counter, IMetricContext context)
+        {
+            this.counter = counter;
+            
+            context.CreateMultiFuncGauge(ProvideMetrics);
+        }
+
+        public static void Measure(LogEventLevelCounter counter, IVostokApplicationMetrics context)
+        {
+            // ReSharper disable once ObjectCreationAsStatement
+            new LogLevelMetrics(counter, context.Instance.WithTag(WellKnownTagKeys.Component, "LogLevel"));
```

Comment:
This tag structure isn't particularly extensible (what if we wanted to add more log metrics later?). My suggestion:
```
Component = VostokLog --> Name = EventsByLevel --> LogLevel = <level>
```